src/ray/raylet/lineage_cache.cc

```diff
@@ -438,7 +458,7 @@ bool LineageCache::ContainsTask(const TaskID &task_id) const {
   return it != entries.end();
 }
 
-size_t LineageCache::NumEntries() const { return lineage_.GetEntries().size(); }
```

Comment:
why not keep this?

Reply:
(or give a method to `Lineage` that provides size?)

Reply:
I added the `GetLineage()` method, and you can get the size using `Lineage::GetEntries().size()`.